<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSnapshot</class>
 <widget class="QDialog" name="DialogSnapshot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snapshot</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxImageSize">
     <property name="title">
      <string>Image Size</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelImageWidth">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxImageWidth">
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelImageHeight">
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxImageHeight">
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelImageScale">
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxImageScale">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxMSAA">
     <property name="title">
      <string>MSAA</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="labelImageSamples">
        <property name="text">
         <string>Samples</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxSamples">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxUseWhiteBackground">
        <property name="text">
         <string>Use white background</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxSaveWindowState">
        <property name="text">
         <string>Save window state</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxExpandFrustum">
        <property name="text">
         <string>Expand frustum if needed</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBoxImageWidth</tabstop>
  <tabstop>spinBoxImageHeight</tabstop>
  <tabstop>doubleSpinBoxImageScale</tabstop>
  <tabstop>spinBoxSamples</tabstop>
  <tabstop>checkBoxUseWhiteBackground</tabstop>
  <tabstop>checkBoxExpandFrustum</tabstop>
  <tabstop>checkBoxSaveWindowState</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonOK</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
